src/redux/slices/hashtagSlices/hashtagsSlice.js

```diff
@@ -64,5 +64,5 @@ export function hashtagsReducer(state = initialState, action) {
   }
 }
 
-export const hashtagsSelect = state => state.hashtags.data;
-export const requestHashtagsSelect = state => state.hashtags.requesting;
+export const hashtagsSelect = state => state.hashtagsReducer.data;
```

Comment:
config state.hashtags rather than hashtagReducer. Reducer. 
"Reducers are functions that take the current state and an action as arguments, and return a new state result."
-> reducer is not the state, careful about naming